packages/grid/_modules_/grid/components/containers/GridRootStyles.ts

```diff
@@ -123,6 +127,8 @@ export const useStyles = makeStyles(
         },
         [`& .${gridClasses.iconButtonContainer}`]: {
           display: 'flex',
+          visibility: 'hidden',
```

Comment:
Why to change the visibility if we have `opacity: 0`?

Reply:
Ah you are right. Only `width: 0` will work.